### DISCUSSION TITLE
A website for example models?

### DISCUSSION THREAD

**Discussion by wang-boyu** [mesa/mesa-examples#61] [Ideas]
I came across with these websites in one of the classes I take this semester: [isee Exchange™](https://exchange.iseesystems.com) and [anylogic cloud](https://cloud.anylogic.com/models). They include models from both the official team as well as the community. Models can be run from within the websites, and there are statistics showing number of runs etc. NetLogo also has its [NetLogo Web](http://www.netlogoweb.org/launch#http://www.netlogoweb.org/assets/modelslib/Sample%20Models/) (but without the statistics). I really like the run statistics as they provide a kind of feedback from users on which models are more popular.

This makes me wonder whether we can have something similar for Mesa example models. One advantage we have over Stella/AnyLogic/NetLogo is that we can rely on Google Colab to run models using Jupyter Notebooks. This would also be in line with what we have for Mesa & Mesa-Geo, where there are readthedocs sites for users and GitHub repos for developers. Similarly there could be a website for example model explorers and this GitHub repo containing all source code of the example models.

**Comment by rht** [comment]
There should be a website with public code that already does this, i.e. a website that showcase a bunch of Jupyter Notebooks. The one I have found so far: https://juliaclimate.github.io/Notebooks/. Not as illustrative as NetLogo's.

**Reply by wang-boyu** [reply]
Looks like this is the source code of the website: https://github.com/JuliaClimate/Notebooks/tree/master/page

It is probably possible to create a similar website (containing a list of all example model notebooks) and host it through github.io. But if we want to somehow track how many times each model is run by users, we would need some extra effort since github pages only serve static content. But maybe simply displaying a list of notebooks is good enough.

**Comment by wang-boyu** [comment]
Raised an issue for this: https://github.com/projectmesa/mesa-examples/issues/139
